torch/_inductor/decomposition.py

```diff
@@ -56,6 +60,14 @@
 )
 decompositions = {**core_aten_decompositions(), **inductor_decompositions}
 
+# Remove unwanted decompositions included via the core ATen decompositions from
+# the Inductor decomp table.
+decomps_to_exclude = [
+    aten._unsafe_index,
+]
+
+remove_decompositions(decompositions, decomps_to_exclude)
```

Comment:
Do we not expect exported models to never be run under inductor? Otherwise this seems like a bad solution.

Reply:
We expect some models to be run under inductor, but we want export to be a generic representation. Export allows users to customize their decompositions (ex. running inductor decomps instead of core aten decomps).